docs/src/examples/components/Avatar/Variations/AvatarExampleGenerateInitials.shorthand.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Avatar } from '@stardust-ui/react'
+
+const generateInitials = (name: string) => {
+  if (!name) {
+    return ''
+  }
+
+  let names = name.split(' ')
+  names = names.filter(item => item !== '')
```

Comment:
You can avoid the `let` and value reassignment by adding the filter to the chained methods:

```jsx
return names
  .filter( ... )
  .map( ... )
  .reduce( ... )
```